.travis.yml

```diff
@@ -15,4 +15,5 @@ install:
   - cp src/config.template.yaml src/config.yaml
 
 script:
-  - pylint --rcfile=pylintrc --extension-pkg-whitelist=alsaaudio,pyA20 src/*.py
+  - pylint --rcfile=pylintrc src/*.py src/alexapi/*.py
```

Comment:
I believe we can squeeze these two lines into just `pylint src/*.py src/alexapi`